database/src/test/scala/za/co/absa/atum/database/runs/CreateOrUpdateAdditionalDataTest.scala

```diff
@@ -38,7 +39,7 @@ class CreateOrUpdateAdditionalDataTest extends DBTestSuite{
       |""".stripMargin
   )
 
-  test("Partitioning and AD present, delete & re-insert, and also 'ignore' of AD records performed") {
+  test("Partitioning and AD present, delete & re-insert, and also 'ignore' of AD records performed", IntegrationTestTag) {
```

Comment:
Why not to tag the whole suite instead of each test. According to http://doc.scalatest.org/2.2.1/index.html#org.scalatest.Tag it should be possible?

Reply:
Done in l latest solution refactoring.

Reply:
Nope, the database module tests are still marked one-by-one. It should be possible to do the whole suite: http://doc.scalatest.org/2.2.1/index.html#org.scalatest.Tag

Reply:
This doc link proposal I was trying before. I tried it again after some time but could not find a working solution.
Using tags at the test method level is still the only way to control the addition/exclusion of these tests.